pkg/cmd/cli/cmd/set/env.go

```diff
@@ -119,6 +130,8 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 	//overwrite := kcmdutil.GetFlagBool(cmd, "overwrite")
 	resourceVersion := kcmdutil.GetFlagString(cmd, "resource-version")
 	outputFormat := kcmdutil.GetFlagString(cmd, "output")
+	from := kcmdutil.GetFlagString(cmd, "from")
+	prefix := kcmdutil.GetFlagString(cmd, "prefix")
```

Comment:
I have said before that I think it's nice to be able to use prefix without from.